src/test/kotlin/dev/stalla/parser/namespace/GoogleplayParserTest.kt

```diff
@@ -88,8 +91,9 @@ internal class GoogleplayParserTest : NamespaceParserTest() {
         node.parseItemChildNodes(builder)
 
         assertThat(builder.googleplayBuilder, "item.googleplay").all {
+            prop(FakeEpisodeGoogleplayBuilder::author).isEqualTo("Lorem Ipsum")
```

Comment:
This reminds me — I wanted to update the test RSS to replace the "lorem ipsum" with unique values (like in the fixtures-dump RSS) so that we also check for reading the right field. It's already somewhat covered by the real-world feed tests, but will do that eventually

Reply:
There is general some room for improvement in certain tests. I've noticed last night that the `ValidatingPodcastItunesBuilderTest` for example is not as thorough as the others.

Reply:
Maybe giving all tests another good look is also something for the 1.0 milestone. Just to increate the confidence that we did everything right.